src/server/destination/another-destination/index.test.js

```diff
@@ -0,0 +1,23 @@
+import { AnotherDestinationExitPage } from './index.js'
+
+describe('#AnotherDestinationController', () => {
+  let page
+
+  beforeEach(() => {
+    page = new AnotherDestinationExitPage()
+  })
```

Comment:
Nothing's mutable here, so can make this `const page = new AnotherDestinationExitPage()`